src/main/resources/doc/fr/html/guide/about/gpl-fr.html

```diff
@@ -384,7 +384,8 @@ <h2>
       <i>Ce programme est distribué dans l’espoir qu’il sera utile, mais SANS AUCUNEGARANTIE : sans même la garantie implicite de COMMERCIALISABILITÉ ni d’ADÉQUATION À UN OBJECTIF PARTICULIER. Consultez la GNU General Public License pour plus de détails.</i>
     </p>
     <p>
-      <i>Vous devriez avoir reçu une copie de la GNU General Public License avec ceprogramme ; si ce n’est pas le cas, consultez : http://www.gnu.org/licenses/</i>
+      <i>Vous devriez avoir reçu une copie de la GNU General Public License avec ceprogramme ; si ce
+        n’est pas le cas, consultez : https://www.gnu.org/licenses/</i>
```

Comment:
That was an automatic change; though I think a maximum line length is a must-have for languages that ignore whitespace such as Java or HTML. `google-java-format` for example limits lines to 100 characters for example (120 is another often used length limit).

Reply:
I totally agree that limiting the maximum line length in source code files using hard line breaks is sensible and should be the preferred approach. Though, for files containing text/documentation, I have made mixed experiences in the past with hard-formatted text files once documents were edited in parallel by different authors and the modifications were merged automatically by Subversion/Git. Due the place for line breaks would frequently vary within a paragraph due to editing and automatic line wrapping, words would sometimes get duplicated or deleted during merging. That's why I have adopted for myself to not use automatic hard line wrapping inside text documents and either leave the whole paragraph on a logic line or start a new line only for each new sentence.

As we don't touch so frequently these HTML files, this might not be such a big issue. We should just make sure they are all consistently formatted (at least within each file) so that potential contributors know which formatting they should follow.